config/locales/es.yml

```diff
@@ -871,7 +871,7 @@ es:
         areas: Ubicaciones
         checkout: Reembolsos y Devoluciones
         configuration: Configuración
-        display_order: Mostrar Pedido
+        display_order: Orden de presentación
```

Comment:
🤔 Not sure about this translation 👉 👈

I preferred the other one. But Spanish is so different depending of the region.

Reply:
Hi 👋 , the problem is that this is not related to Orders but to the order with which items are displayed on the screen. The previous translation was literally implying that this meant "displaying an order" which is wrong. 

Out of context is hard to spot this one, I noticed it on the Admin Panel, it simply didn't make sense.

Reply:
I agree with @manuca . I think that the translation makes sense as the menu makes reference to the order in which product within a taxon are displayed on listing pages and search.

Reply:
Gotcha!

In that case, totally make sense. Thank you so mucho for clear this doubt 🙌 

Thank you 😄